examples/status-message/src/lib.rs

```diff
@@ -21,11 +21,7 @@ impl StatusMessage {
         self.records.insert(account_id, message);
     }
 
-    pub fn get_status(&self, account_id: String) -> Option::<String> {
-        assert!(
-            env::is_valid_account_id(account_id.as_bytes()),
-            "Given account ID is invalid"
-        );
+    pub fn get_status(&self, account_id: AccountId) -> Option::<String> {
```

Comment:
```suggestion
    pub fn get_status(&self, account_id: AccountId) -> Option<String> {
```

Or is the turbofish actually required by the macro?

Reply:
It is required, I haven't dug into why specifically it's needed yet

Reply:
it's needed because of the metadata macro generation, because it calls `Option::<String>::schema_container()`